### PR TITLE
Move publish_distro 817e024 into openSUSE-release-tools

### PR DESCRIPTION
**Pull request by lkocman**
Suggested in https://github.com/openSUSE/publish_distro/issues/13 to separate tool and configuration.

### CODE REVIEW

**Comment by lkocman** [comment]
https://github.com/openSUSE/publish_distro/pull/21 (removal)

**Comment by ancorgs** [comment]
First of all, please add the description of this new tool at https://github.com/openSUSE/openSUSE-release-tools/blob/master/CONTENTS.md as part of this pull request.

The main problem with this repository is that it has been historical just a bunch of poorly documented scripts (which doesn't even describe their function when executed without arguments), thrown into the repository by different people via pull requests with one-liner descriptions.

The file CONTENTS.md is at least a poor attempt to fix that by providing some context so we can know what a script is used for (and by who), where is it deployed and whether we can remove it from the repository at a given point of time.

Talking about deployment. Is this expected to become part of https://build.opensuse.org/package/show/openSUSE:Tools/openSUSE-release-tools ?

**Comment by coolo** [comment]
And shouldn't all those configs move too from that repo? If so, better move it to a subdirectory, add a README there explaining (and point to that from CONTENT.md as @ancorgs mentions) and if you plan to deploy it on pontifex, add it to the package as well.

**Comment by DimStar77** [comment]
> @DimStar77 would such proposal be okay with you?

If we want to consolidate this into a single repo, then it does make sense to move it all together

Question is if we win anything over having it in a sep repo

**Comment by lkocman** [comment]
The original bonus was that we consume tool in a more controlled way.

**Comment by coolo** [comment]
I personally don't care either way - you can also package the other repo. But if you want it to be in this repo, it needs to follow the rules

**Comment by dirkmueller** [comment]
@lkocman the underlying issue should be fixed now, can you please rebase?

**Comment by coolo** [comment]
This still lacks packaging - I thought that was the plan to begin with

File: dist/package/openSUSE-release-tools.spec [hunk 1]
```diff
@@ -187,6 +187,19 @@ BuildArch:      noarch
 %description origin-manager
 Tools for managing the origin of package sources and keeping them in sync.
 
+%package publish-distro
+Summary:        Tool for publishing ftp-stage to ftp-prod
+Group:          Development/Tools/Other
+Requires:       rsync
+# Currently in openSUSE infra repo
+Requires:       python-mb
```
Comment (coolo) [inline dist/package/openSUSE-release-tools.spec:195]: python-? that sounds like python2. Is that true?

Comment (coolo) [reply dist/package/openSUSE-release-tools.spec:195]: All of the mb calls seem to be commented

**Review by coolo — APPROVED** [review]
Good enough for me. I didn't look at the script itself - I did that mistake some years ago... :)

**Comment by lkocman** [comment]
Hehe thank you @coolo :-) Merging

**Pull request merged**